### ISSUE TITLE
Should it not use hub apiserver host as the managed cluster?

### ISSUE THREAD

**Issue by wangglbj** [open-cluster-management-io/clusteradm#76]
I found after using clusteradm join new cluster to hub cluster. it use the hub apiserver host as the managed cluster. and then I checked the clusteradm code,  is the cluster ip the managed cluster external server ip/host, right? if it's right, it need user to specify it, as shown in the following example:

#kubectl get managedcluster
NAME       HUB ACCEPTED   MANAGED CLUSTER URLS         JOINED   AVAILABLE   AGE
cluster1        true                    https://hub apiserver host:6443     True       True           21m
cluster2       true                    https://hub apiserver host:6443     True       True           64s

External server host, [klusterlets.cr.yaml](https://github.com/open-cluster-management-io/clusteradm/blob/main/pkg/cmd/join/scenario/join/klusterlets.cr.yaml#L11) is it not correct ?

**Comment by qiujian16** [comment]
yes, I think it is not correct

/kind bug

**Comment by xauthulei** [comment]
@wangglbj , This bug have been fixed in since `clusteradm` v1.0 https://github.com/open-cluster-management-io/clusteradm/blob/630dd18c4684932caac453a418e9ff70631707b7/pkg/cmd/join/scenario/join/klusterlets.cr.yaml#L12-L14

In the latest version it is OK. e.g.(my cluster `llhu-1`):
```
llhu@huleis-mbp ~ % kubectl get mcl 
NAME     HUB ACCEPTED   MANAGED CLUSTER URLS                JOINED   AVAILABLE   AGE
llhu-1   true           https://llhu-1-control-plane:6443   True     Unknown     21h
```
You can close this issue, Thanks
